Makefile

```diff
@@ -1,4 +1,21 @@
 #! /usr/bin/make
 
-generated-%.js: tools/generate-js.py
-	(echo '// Generated by $<: DO NOT EDIT'; tools/generate-js.py $*) > $@
+SOURCE := $(wildcard javascript/*.js) $(wildcard python/*.py)
+
+check: check-python check-javascript check-quotes
+
+check-quotes: $(SOURCE:%=check-quotes/%) $(SOURCE:%=check-quotes/%)
+
+check-quotes/%.py: %.py
+	@echo $@
+	@tools/check_quotes.py --comment-start='# ' --comment-continue='#' --boltdir=lightning-rfc/ $<
+
+check-quotes/%.js: %.js
+	@echo $@
+	@tools/check_quotes.py --comment-start='// ' --comment-continue='//' --boltdir=lightning-rfc/ $<
```

Comment:
should we add '/* .. */' for long comments?

Reply:
Currently you have to define a single format; as long as we're consistent with how we quote the BOLTs, I guess this is OK?